src/databricks/labs/ucx/queries/migration/main/04_1_remaining_hms_tables.sql

```diff
@@ -0,0 +1,6 @@
+/* --title 'List of remaining tables in HMS' --width 6 */
```

Comment:
A query with one column is automatically rendered as a counter. Make sure this query is rendered as a table by explicitly setting the type
```suggestion
/* --title 'List of remaining tables in HMS' --type table --width 6 */
```

Please share another screenshot of the dashboard after updating